templates/tftmpl/root_test.go

```diff
@@ -170,6 +173,125 @@ func TestAppendRootProviderBlocks(t *testing.T) {
 	}
 }
 
+func TestAppendRootModuleBlocks(t *testing.T) {
+	workingDir, err := os.Getwd()
+	assert.Nil(t, err, "Error determining current working directory")
```

Comment:
```suggestion
	require.Nil(t, err, "Error determining current working directory")
```

One minor nit is to change this assertion to require since the rest of the test would never pass if this had errored. `assert` continues running through the rest of the test even if it fails.